### PR TITLE
feat: add folders and project descriptions

### PR DESCRIPTION
**Pull request by pedromsmoreira**
Basic structure of the projects

Ideas for implementation

Project Objectives in order to give some flavor to it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,37 @@
...
+Create a system that allow someone to search for pets that need adoption.
+This system should also provide an easy access to a Veterinarians and Pet Hospitals Network, making it simple to share data between them.
+This system should also allow pets to be advertised for adoption and where they are and their conditions. Mainly make things simple enough to facilitate the adoption of abandoned pets.
+Register former pets ouwner who abandoned a pet and place them in a blacklist. This will prevent pets from being placed/adopted by owners who won't really care about them.
+
+## Repository structure
+
+- backend-api -> will have the base api for the system to work
+- frontend -> will have the frontend application that will call the backend api
+- workder-services -> will have the background jobs that will manipulate, load and ingest data from both backend and frontend
```
Comment (joao-r-reis) [inline README.md]:
```suggestion
- worker-services -> will have the background jobs that will manipulate, load and ingest data from both backend and frontend
```

File: worker-service/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# pms
+
+Pet Management System
+
+## workder-services
+
+- workder-services -> will have the background jobs that will manipulate, load and ingest data from both backend and frontend
```
Comment (joao-r-reis) [inline worker-service/README.md]:
```suggestion
- worker-services -> will have the background jobs that will manipulate, load and ingest data from both backend and frontend
```

File: worker-service/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# pms
+
+Pet Management System
+
+## workder-services
```
Comment (joao-r-reis) [inline worker-service/README.md]:
```suggestion
## worker-services
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,37 @@
...
+- infrastructure-services -> will have application or script that will help manage infrastructure (TBD: not sure if it worth)
```
Comment (joao-r-reis) [inline README.md]: Maybe leave this out for now until we know what we want this to be?

**Review by joao-r-reis — APPROVED** [review]
:+1: cool idea

File: README.md (hunk 4 above)
Comment (pedromsmoreira) [reply README.md]: I agree I will mark it as something without priority or no need for now.

**Pull request merged**